tests/run.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
 
-py.test -s --tb=short -vv /testing/tests "$@"
+pip install pytest-golden
```

Comment:
This should be in the `Dockerfile` instead.

Reply:
```
    image: mendersoftware/mender-test-containers:acceptance-testing
```
So I would have to add it to this image, which I decided against, considering the use case. But if you insist.

Reply:
Okey, I don't insist :)